addons/packages/sriov-network-device-plugin/3.3.2/README.md

```diff
@@ -0,0 +1,43 @@
+# Sriov-Network-Device-Plugin Package
+
+This package contains Kubernetes device plugin for discovering and advertising SR-IOV virtual functions (VFs) available on a Kubernetes host. Using [sriov-network-device-plugin](https://github.com/k8snetworkplumbingwg/sriov-netowrk-device-plugin/).
+
+## Components
+
+* SRIOV Network Device Plugin DaemonSet
+* SRIOV Network Device Plugin ConfigMap
+* SRIOV Network Device Plugin ServiceAccount
+
+## Configuration
+
+The following configuration values can be set to customize the sriov-network-device-plugin installation.
+
+### Global
+
+| Value       | Required/Optional | Description                                                   |
+| ----------- | ----------------- | ------------------------------------------------------------- |
+| `namespace` | Required          | The namespace in which to deploy sriov-network-device-plugin. |
+
+### sriov-network-device-plugin DaemonSet Configuration
+
+| Value                   | Required/Optional | Description                                                                                                     |
+| ----------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------- |
+| `daemonset.resources`             | Required          | The resources daemonset settings including resources. |
```

Comment:
The description could be improved like "The requests and limits daemonset requires"

Reply:
Updated